Runtime/Scripts/BaseMeshSync.cs

```diff
@@ -1239,6 +1239,9 @@ private EntityRecord UpdateMeshEntity(MeshData data, MeshSyncPlayerConfig config
 
                 meshUpdated = true;
             }
+            else if (rec.mesh != null) {
+                rec.mesh.Clear();
+            }
```

Comment:
This will be called when we remove all points and indices from a Mesh we have already synched?